public/gallery.html

```diff
@@ -106,12 +132,180 @@
 			<button class="bg-black text-white text-lg ring-2 py-1 px-4 rounded-lg self-center" type="submit"><strong>Apply</strong></button>
 		</div>
 	</div>
-<!-- End filters container -->
 </div>
+<!-- End filters container -->
+
 <!-- Menu end -->
-<div>
-	<img  src="Assets/images/gallery/percare_gallery_hero.png" alt="Animals posing in a art gallery">
+
+<!-- Grid section -->
+<div id="backgroundImg">
+
+<!-- Grid container start -->
+	<div class="w-full h-full grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-4" id="imgContainer">
```

Comment:
So the entire website will only ever have a single imgContainer? 

This is what results from the combination of name and id. 

imgContainer is generic. There could be many, multiple image containers in a page and in an entire website. 
So this name would be more appropriate as a class. 

So you could have multiple elements with the class imgContainer and then each element will have its own id which differences through its purpose. In this case, petGallery might be an idea. 

Now that I think about it, this element contains a bunch of images. imgContainer usually means "image container" so this leads me to a single image. Not multiple. 

I see e changes required here